platform/engineering/official_github_teams_process

```diff
@@ -0,0 +1,38 @@
+# GitHub Teams on for VA.gov
+
+## What is GitHub Teams, and why are we talking about it?
+
+GitHub Teams give organizations the ability to create groups of members and control access to repositories. Team members can be granted read, write, or admin permissions to specific repositories.
+For more information on GitHub teams, please see [GitHub docs](https://help.github.com/en/enterprise/2.19/admin/user-management/creating-teams).
+
+At the moment, some teams have their own GitHub Teams, while others do not. GitHub Teams’ benefits and functions are not fully understood across VA.gov. Our goal is to provide a basic understanding of GitHub teams, establishing a logical structure of teams first. Then we can improve the process of granting access and developers’ experience on VA.gov. 
+
+In the future, we will be using *Official GitHub Teams* as a source of truth for team rosters, access, and auditing purposes for VSP. The process of creating and maintaining Official GitHub Teams is below. 
+
```

Comment:
I dislike the term "Official" here, but don't have a suggestion on an alternative yet. What are we trying to describe in calling it that? "A team we actually use for permissions"? I think the distinction we are trying to convey here is that because anyone can name a team whatever they want, the systems in place won't pick up and use it just because it matches a pattern, right?

Reply:
Yes, and also so we know which are teams and which are not in general. Maybe using the word "recognized"?

Reply:
Verified. approved?

Reply:
changed to verified